diff --git a/buildSrc/electron-package-json-template.js b/buildSrc/electron-package-json-template.js
index 5fc9e2706..dfc051d69 100644
--- a/buildSrc/electron-package-json-template.js
+++ b/buildSrc/electron-package-json-template.js
@@ -72,6 +72,7 @@ module.exports = function (opts) {
 			"keytar": pj.dependencies.keytar
 		},
 		"build": {
+			"npmArgs": ["--cache=/run/build/tutanota/npm-cache"],
 			"electronVersion": pj.devDependencies.electron,
 			"icon": iconPath,
 			"appId": "de.tutao.tutanota" + nameSuffix,
@@ -161,7 +162,7 @@ module.exports = function (opts) {
 				"target": [
 					{
 						"target": unpacked ? "dir" : "AppImage",
-						"arch": "x64"
+						"arch": (process.arch === "arm") ? "armv7l" : process.arch
 					}
 				]
 			}
