diff --git a/buildSrc/DesktopBuilder.js b/buildSrc/DesktopBuilder.js
index ad2b0f5cd..31233a62b 100644
--- a/buildSrc/DesktopBuilder.js
+++ b/buildSrc/DesktopBuilder.js
@@ -86,11 +86,9 @@ function build(opts) {
 				)
 			)
 		}).then(() => Promise.all([
-			fs.removeAsync(path.join(distDir, '/installers/')),
 			fs.removeAsync(path.join(distDir, '/node_modules/')),
 			fs.removeAsync(path.join(distDir, '/cache.json')),
 			fs.removeAsync(path.join(distDir, '/package.json')),
-			fs.removeAsync(path.join(distDir, '/package-lock.json')),
 			fs.removeAsync(path.join(distDir, '/src/')),
 		]))
 }
diff --git a/buildSrc/electron-package-json-template.js b/buildSrc/electron-package-json-template.js
index 5fc9e2706..d78e7e9ce 100644
--- a/buildSrc/electron-package-json-template.js
+++ b/buildSrc/electron-package-json-template.js
@@ -72,6 +72,7 @@ module.exports = function (opts) {
 			"keytar": pj.dependencies.keytar
 		},
 		"build": {
+			"npmArgs": ["--cache=/run/build/tutanota/npm-cache"],
 			"electronVersion": pj.devDependencies.electron,
 			"icon": iconPath,
 			"appId": "de.tutao.tutanota" + nameSuffix,
@@ -161,7 +162,7 @@ module.exports = function (opts) {
 				"target": [
 					{
 						"target": unpacked ? "dir" : "AppImage",
-						"arch": "x64"
+						"arch": (process.arch == "arm")?"armv7l":process.arch
 					}
 				]
 			}
diff --git a/dist.js b/dist.js
index b7d44df61..cfc08b339 100644
--- a/dist.js
+++ b/dist.js
@@ -101,8 +101,7 @@ function measure() {
 }
 
 function clean() {
-	return fs.removeAsync("build")
-	         .then(() => fs.ensureDirAsync(DistDir + "/translations"))
+	return fs.ensureDirAsync(DistDir + "/translations"))
 }
 
 function buildWebapp() {
