diff --git a/buildSrc/DesktopBuilder.js b/buildSrc/DesktopBuilder.js
index ad2b0f5cd..4efab506f 100644
--- a/buildSrc/DesktopBuilder.js
+++ b/buildSrc/DesktopBuilder.js
@@ -90,7 +90,6 @@ function build(opts) {
 			fs.removeAsync(path.join(distDir, '/node_modules/')),
 			fs.removeAsync(path.join(distDir, '/cache.json')),
 			fs.removeAsync(path.join(distDir, '/package.json')),
-			fs.removeAsync(path.join(distDir, '/package-lock.json')),
 			fs.removeAsync(path.join(distDir, '/src/')),
 		]))
 }
